pyproject.toml

```diff
@@ -1,7 +1,121 @@
 [build-system]
-requires = ["setuptools >= 40.6.0", "wheel", "setuptools_scm>=6.2"]
+requires = [
+    "setuptools >= 40.6.0",
+    "setuptools_scm>=6.2",
+]
```

Comment:
just as a minor remark - why did we need wheel when we never built wheels for esmvalcore?

Reply:
We do build wheels, check PyPI: https://pypi.org/project/ESMValCore/#files

However, it does not need to be listed as a dependency according to Repo-Review/[Scientific Python Library Development Guide](https://learn.scientific-python.org/development/guides/packaging-classic/#pep-517518-support-high-priority):

> Note that "wheel" is never required; it is injected automatically by setuptools only when needed.

Reply:
https://github.com/ESMValGroup/ESMValCore/blob/12798741cd4422adc1b7a6ecbc987dbc6e488388/.github/workflows/build-and-deploy-on-pypi.yml#L35-L41

Reply:
Thanks! Funny that we build wheels even if we don't require compilation, but maybe what I'm callin a "wheel" is the wrong thing here :wheel: All good!